src/VirtualClient/VirtualClient.Dependencies/BlenderInstalltion.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Dependencies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO.Abstractions;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.DependencyInjection;
+    using Polly;
+    using VirtualClient.Common;
+    using VirtualClient.Common.Extensions;
+    using VirtualClient.Common.Telemetry;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    /// Installation component for Blender Benchmark cli.
+    /// </summary>
+    public class BlenderInstallation : VirtualClientComponent
```

Comment:
Can this be merged into the executor's Initialize step? Generally prefer not to have separate dependency if it is workload specific but could make exception if makes sense